dwave/cloud/config.py

```diff
@@ -724,21 +722,21 @@ def load_config(config_file=None, profile=None, client=None,
             Config file parse failed.
 
     Examples
-        Additional examples are given in :mod:`dwave.cloud.config`.
+        This example loads the configuration from an auto-detected configuration file
+        in the home directory of a Windows system user.
+
+        >>> import dwave.cloud as dc
+        >>> dc.config.load_config()
```

Comment:
Can we avoid the awful `dc` module name?
`from dwave.cloud.config import load_config, get_configfile_paths`, or `from dwave.cloud import config`?

Reply:
Yes, I've been slowly hunting these down. I'll do that next time I'm in this repo, I'll get the current fixes in for you now.